app/static/src/app/utils.ts

```diff
@@ -176,4 +176,18 @@ export function generateBatchPars(
     };
 }
 
-export const newSessionId = () => uid(32);
+export const newSessionId = (): string => uid(32);
+
+export const joinStringsSentence = (strings: string[], last = " and ", sep = ", "): string => {
```

Comment:
A [poem](https://brianbilston.com/2015/07/27/the-importance-of-the-oxford-comma/) in support of the Oxford Comma for your consideration. 😂 Although this would likely cause a headache for when there are only two errors.